src/app/src/pages/Mainpage.tsx

```diff
@@ -150,7 +134,10 @@ export default function Mainpage({ currentUser }: Props) {
                 id={post.id}
               />
             ))}
-            </List>
+          </List>
+          <GitContainer>
+            <Github username={currentUser?.displayName} />
```

Comment:
A user's display name is not necessarily their github username. Authors have a 'github' attribute that is the full url to their github profile, from the spec:
` # HATEOS url for Github API`
`"github": "http://github.com/laracroft",`
Display names seem to be just for display purposes, not like an actual username.